placement-rules-in-sql.md

```diff
@@ -149,22 +152,30 @@ CREATE TABLE t1 (
 You can directly attach the default placement options to a database schema. This works similar to setting the default character set or collation for a schema. Your specified placement options apply when no other options are specified. For example:
 
 ```sql
+CREATE PLACEMENT POLICY p1 PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2";  -- Create placement policies
+
+CREATE PLACEMENT POLICY p2 FOLLOWERS=4;
+
+CREATE PLACEMENT POLICY p3 FOLLOWERS=2;
+
 CREATE TABLE t1 (a INT);  -- Creates a table t1 with no placement options.
 
-ALTER DATABASE test FOLLOWERS=4;  -- Changes the default placement option, and does not apply to the existing table t1.
+ALTER DATABASE test POLICY=p2;  -- Changes the default placement option, and does not apply to the existing table t1.
+
+CREATE TABLE t2 (a INT);  -- Creates a table t2 with the default placement policy p2.
 
-CREATE TABLE t2 (a INT);  -- Creates a table t2 with the default placement of FOLLOWERS=4.
+CREATE TABLE t3 (a INT) POLICY=p1;  -- Creates a table t3 without the default policy p2, because this statement has specified another placement policy.
 
-CREATE TABLE t3 (a INT) PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2";  -- Creates a table t3 without the default FOLLOWERS=4 placement, because this statement has specified another placement.
+ALTER DATABASE test POLICY=p3;  -- Changes the default policy, and does not apply to existing tables.
 
-ALTER DATABASE test FOLLOWERS=2;  -- Changes the default placement, and does not apply to existing tables.
+CREATE TABLE t4 (a INT);  -- Creates a table t4 with the default policy p3.
 
-CREATE TABLE t4 (a INT);  -- Creates a table t4 with the default FOLLOWERS=2 option.
+ALTER PLACEMENT POLICY p3 FOLLOWERS=3; -- Tables with policy p3, i.e. t4, will have FOLLOWERS=3。
 ```
 
-Because placement options are only inherited from the database schema when a table is created, it is recommended to set the default placement option using a `PLACEMENT POLICY`. This ensures that future changes to the policy propagate to existing tables.
+One could use [ALTER PLACEMENT POLICY](/sql-statements/sql-statement-alter-placement-policy.md) to change a policy, and changes will be propagated to all objects with the corresponding policy.
 
-### Advanced placement
+### Advanced placement option
 
```

Comment:
Line 214: placement policies -> placement rules, according to https://github.com/pingcap/docs-cn/pull/8230/files#diff-5e2b3452de0170d2f05486307b1a3201e0ac80c785f89651f25fb53db5419b6aR213

Line 216: placement options -> placement rules, according to  https://github.com/pingcap/docs-cn/pull/8230/files#diff-5e2b3452de0170d2f05486307b1a3201e0ac80c785f89651f25fb53db5419b6aR215

Reply:
What?